backend/src/core/services/job.service.ts

```diff
@@ -123,9 +141,36 @@ const retryCampaign = (campaignId: number): Promise<any> | undefined => {
   })
 }
 
+const updateScheduledCampaign = async (
```

Comment:
would prefer naming this to make clear it is updating the `scheduledTime` of a campaign

Reply:
e.g. `updateCampaignScheduledTime` or sth

Reply:
scheduledTime is a property of the campaign, so you are updating the campaign. Also see the rest of the function names in this file, it is to be consistent.